src/stylesheets/scss/_layout.scss

```diff
@@ -65,6 +65,15 @@
 }
 .preheader {
   display: none !important;
+  visibility: hidden;
+  mso-hide: all;
+  font-size: 1px;
+  color: #ffffff;
```

Comment:
Is specifying a color necessary? It seemed to do fine without this property when I litmus tested it.